src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -701,14 +701,15 @@ internal unsafe struct MEMORY_BASIC_INFORMATION {
         internal const String OLE32    = "ole32.dll";
         internal const String OLEAUT32 = "oleaut32.dll";
         internal const String NTDLL    = "ntdll.dll";
+        internal const String ADVAPI32 = "advapi32.dll";
```

Comment:
I suspect that this will need a bunch of `AppDomain.IsAppXModel` checks to restrict the implementation for UWP.